harness/determined/common/experimental/user.py

```diff
@@ -5,73 +5,63 @@
 
 
 class User:
-    def __init__(
-        self,
-        user_id: int,
-        username: Optional[str],
-        admin: Optional[bool],
-        session: api.Session,
-        active: Optional[bool] = True,
-        remote: Optional[bool] = False,
-        display_name: Optional[str] = None,
-        agent_uid: Optional[int] = None,
-        agent_gid: Optional[int] = None,
-        agent_user: Optional[str] = None,
-        agent_group: Optional[str] = None,
-    ):
-        self.username = username
-        self.admin = admin
+    def __init__(self, user_id: int, session: api.Session):
```

Comment:
A docstring would be nice here, too. Again, doesn't need to be this PR's job. But maybe if you choose to punt, could you please create a ticket to help us remember?

Reply:
definitely missing a docstring. updated.

Reply:
Thanks! Resolving this in favor of detailed questions.